perfkitbenchmarker/linux_benchmarks/object_storage_service_benchmark.py

```diff
@@ -589,9 +618,10 @@ def Prepare(self, vm):
     vm.PushFile(FLAGS.boto_file_location, DEFAULT_BOTO_LOCATION)
 
     vm.bucket_name = 'pkb%s' % FLAGS.run_uri
+    region = FLAGS.object_storage_region or DEFAULT_AWS_REGION
 
     vm.RemoteCommand(
-        'aws s3 mb s3://%s --region=us-east-1' % vm.bucket_name)
+        'aws s3 mb s3://%s --region=%s' % (vm.bucket_name, region))
```

Comment:
what happens if the region is invalid?

I presume an exception will be thrown (vm.RemoteCommand would fail?) and prepare stage would fail.

can you try it quickly to confirm this...